### PR TITLE
Hammerdb

### PR DESCRIPTION
**Pull request by ebattat**
### Description
PVC, Hostpath and Local support for hammerdb
### Fixes
Adding the following support for VM and Pod:
1. OCS PVC (Pod and VM) - Openshift Container Storage
2. Hostpath (Pod and VM) - Local OS disk
3. Local (POD only) - Local separate disk  

Adding related examples under hammerdb_crds.

### CODE REVIEW

**Review by jtaleric — COMMENTED** [review]
Some references to your personal repo here.

**Comment by ebattat** [comment]
Hi,

Do you mean to image reference ?

**Comment by ebattat** [comment]
Hi,

I have updated all the database container disks into quay.io/cloud-bulldozer
.
https://github.com/cloud-bulldozer/benchmark-operator/pull/596/commits/97f90b6331d62dd1280285fcc01c69373e973c7d

Best Regards,
Eli Battat.

File: roles/hammerdb/tasks/main.yaml [hunk 1]
```diff
@@ -25,24 +25,26 @@
...
-          volume.beta.kubernetes.io/storage-class: "{{ workload_args.client_vm.storageclass }}"
-      spec:
-        accessModes:
-          - "{{ workload_args.client_vm.pvcaccessmode | default('ReadWriteOnce') }}"
-        volumeMode: "{{ workload_args.client_vm.pvcvolumemode | default('Filesystem') }}"
-        resources:
-          requests:
-            storage: "{{ workload_args.client_vm.storagesize }}"
-  when: workload_args.client_vm.storageclass is defined
+- block:
```
Comment (rsevilla87) [inline roles/hammerdb/tasks/main.yaml:28]: Why having a block with a single task?

Comment (ebattat) [reply roles/hammerdb/tasks/main.yaml]: Fixed: https://github.com/cloud-bulldozer/benchmark-operator/pull/596/commits/ee4b09e713f1991e8369c0032053a65110bf2f12

**Comment by rsevilla87** [comment]
Hey @ebattat, mind you update hammerdb docs with the new variables.

**Comment by ebattat** [comment]
> Hey @ebattat, mind you update hammerdb docs with the new variables.

Fixed https://github.com/cloud-bulldozer/benchmark-operator/pull/596/commits/648b80599f53be725bd01fab33501f10801ee7ea

File: resources/crds/hammerdb_crds/mssql/ripsaw_v1alpha1_hammerdb_mssql_vm.yaml [hunk 2]
```diff
@@ -90,10 +90,16 @@ spec:
...
+        pvc_pvcaccessmode: ReadWriteMany
         # Can be one of Filesystem,Block Default: Filesystem
-        #pvcvolumemode: Block
-        #storagesize: 5Gi
+        pvc_pvcvolumemode: Block
+        pvc_storagesize: 10Gi
+        ## HostPath - Configuring SELinux on cluster workers
+        hostpath: false # enable for hostpath
+        hostpath_path: /var/tmp/disk.img
+        hostpath_storagesize: 10Gi
```
Comment (rsevilla87) [inline resources/crds/hammerdb_crds/mssql/ripsaw_v1alpha1_hammerdb_mssql_vm.yaml:104]: Hey @ebattat  I'm still missing some documentation for all these fields

Comment (ebattat) [reply resources/crds/hammerdb_crds/mssql/ripsaw_v1alpha1_hammerdb_mssql_vm.yaml]: Done https://github.com/cloud-bulldozer/benchmark-operator/pull/596/commits/94e03e13df29554a918f2ca50172400a23cd032c

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Review by mkarg75 — APPROVED** [review]
lgtm

**Comment by rsevilla87** [comment]
@jtaleric we need your lgtm stamp to merge this one

**Pull request merged**